torch/nn/modules/conv.py

```diff
@@ -235,13 +236,13 @@ class Conv2d(_ConvNd):
 
     .. note::
 
-         The configuration when `groups == in_channels` and `out_channels == K * in_channels`
+         The configuration when ``groups == in_channels`` and ``out_channels == K * in_channels``,
```

Comment:
also change them at `conv1d`

Reply:
also, it is hard to understand what it mean at first glance. Maybe rephrase to "It is also know as `depwise convolution` when `group == in_channels` and `out_channels == K \star in_channels`, where `K` is a positive integer"